tests/test_data.yml

```diff
@@ -57,3 +57,13 @@ pinterest:
     - "steve"
   available_usernames:
     - "zNRe3jx3isA8CoM"
+openhub:
+  taken_usernames:
```

Comment:
can you add 2 more taken usernames for both these websites?

Reply:
sure